### ISSUE TITLE
Finite (non-instantaneous) smelting times

### ISSUE THREAD

**Issue by deathcap** [deathcap/voxel-furnace#3]
Have the furnace take some time to smelt, instead of doing it instantly. Probably consume the item into an internal buffer, then output the result after some time.

**Comment by z3t0** [comment]
Maybe we could have certain fuels have properties of energy density and max burn time which can then be used to calculate time required to cook something

**Comment by deathcap** [comment]
Yeah those could be properties in voxel-registry for each item (could be as straightforward as "seconds required to smelt", or calculated from other properties)

**Comment by z3t0** [comment]
ok, I'll take a look at that once I'm done going through the list of plugins so I understand them all
